### PR TITLE
refactor: drop superorder from metas

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
No more needing to remember what the superOrder for a given superBlock is.

### CODE REVIEW

File: curriculum/utils.js [hunk 1]
```diff
@@ -16,3 +16,32 @@ exports.testedLang = function testedLang() {
...
+  'machine-learning-with-python': 10,
+  'coding-interview-prep': 11
+};
+
+function getSuperOrder(superblock, { isLegacy } = { isLegacy: false }) {
+  if (typeof superblock !== 'string')
+    throw Error('superblock must be a string');
+  const order = superBlockToOrder[superblock];
+  if (typeof order === 'undefined')
+    throw Error(`${superblock} is not a valid superblock`);
```
Comment (nhcarrigan) [inline curriculum/utils.js:43]: This is throwing on the `certifications` superblock.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/utils.js (hunk 1 above)
Comment (ojeytonwilliams) [reply curriculum/utils.js:43]: Hmm.  Do we want to give that a `null` `superOrder`?

Comment (nhcarrigan) [reply curriculum/utils.js:43]: Or filter the test to not care about the `certifications` superblock.

Comment (ojeytonwilliams) [reply curriculum/utils.js:43]: Could do.  Since `superOrder` doesn't really mean anything for `certifications` probably safest to never set it, though.

**Comment by nhcarrigan** [comment]
https://github.com/freeCodeCamp/freeCodeCamp/blob/c98728ccd0619b62ec43bc31ba5c69384063253c/curriculum/test/test-challenges.js#L274

So removing the superOrder effectively disables this test. I removed the filter from this line locally and the tests pass - I can push a commit if you'd like.

**Comment by ojeytonwilliams** [comment]
Sure, commit away.

**Review by naomi-lgbt — DISMISSED** [review]

**Comment by nhcarrigan** [comment]
@ojeytonwilliams I gave this a rebase to resolve the conflicts with deleting the certification metas.

File: curriculum/utils.js [hunk 2]
```diff
@@ -16,3 +16,33 @@ exports.testedLang = function testedLang() {
...
+  'data-visualization': 3,
+  'relational-databases': 4,
+  'back-end-development-and-apis': 5,
+  'quality-assurance': 6,
+  'scientific-computing-with-python': 7,
+  'data-analysis-with-python': 8,
+  'information-security': 9,
+  'machine-learning-with-python': 10,
+  'coding-interview-prep': 11,
+  certifications: null
```
Comment (nhcarrigan) [inline curriculum/utils.js:36]: So I think we can revert all of this logic now that certifications don't have a meta.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/utils.js (hunk 2 above)
Comment (ojeytonwilliams) [reply curriculum/utils.js:36]: Just drop the 'certifications' special cases? Great, that'll be much nicer.

Comment (nhcarrigan) [reply curriculum/utils.js:36]: Right because we no longer have certification metas at all, right?

Comment (ojeytonwilliams) [reply curriculum/utils.js]: Sorry, just saw this. Yes, exactly. 

Somehow despite working on both PRs I didn't put 2 and 2 together...

**Review by moT01 — APPROVED** [review]
I'm good with this - LGTM :tada:

**Review by naomi-lgbt — APPROVED** [review]
Okay I think I'm happy with this.

**Pull request merged**